src/status_im/utils/http.cljs

```diff
@@ -56,3 +55,6 @@
        (.catch (or on-error
                    (fn [error]
                      (utils/show-popup "Error" (str error))))))))
+
+(defn match-url [url]
```

Comment:
Maybe we could improve the name of this fn? Like `normalize-url` ? (I know it was there already :))

Reply:
yes, i agree, `normalize-url` sounds better, thanks